packages/vue/src/components/atoms/SfInput/SfInput.stories.js

```diff
@@ -56,21 +56,22 @@ storiesOf("Atoms|Input", module)
         value: "",
       };
     },
-    template: `<div style="padding: 10px">
+    template: `
+      <div style="padding: 10px">
       <SfInput
-        v-model="value"
-        :type="type"
-        :label="label"
-        :name="name"
-        :valid="valid"
-        :error-message="errorMessage"
-        :required="required"
-        :disabled="disabled"
-        :has-show-password="hasShowPassword"
-        :class="customClass"
-        :placeholder="placeholder"
+          v-model="value"
```

Comment:
Please, remove these additional spaces, we don't need them :)

Reply:
This are due to indentation @justyna-13

Reply:
I have anyways made the changes, please can I create another PR for this issue instead of this one so that it can be counted in hacktoberfest??